source/modules/assistance/categories.rst

```diff
@@ -0,0 +1,65 @@
+Categories
+==========
+
+A category can be made hierarchical:
+
+* by selecting a parent category using in first tab the field `As child of`
+* by adding a new child category in tab `ITIL Categories`
+
+Ticket categories
+-----------------
+
+Ticket categories can be managed in dropdown setup *Setup > Dropdowns > Ticket Categories*.
+
+Ticket categories are also used by changes and problems.
+
+.. image:: images/categorieTicket.png
+   :alt: Ticket category
+   :align: center
+
+A person in charge and/or a technical group can be added to this category and will then be used for notifications.
+
+A knowledge base category can also be associated to this category; this will register the solution in this category of the knowledge base.
+
+.. ??? ***Exemple :*** - catégorie de ticket : panne disque dur - champ Base de connaissance de la catégorie : Panne > Ordinateur Si la catégorie "Panne disque dur" est sélectionnée dans le ticket, lors de l'ajout de la solution du ticket dans la base de connaissance, celle-ci sera stockée dans "Panne > Ordinateur".
```

Comment:
In this example, if you have a ticket with the category "Hard disk failure" which is linked to the KB category "Failure > Computer", when you add a solution to the ticket with "Save to Knowledge base" set to "Yes", the default category in the new Knowledge base Article form that opens will be "Failure > Computer".

Reply:
Integrated!